<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditDialog</class>
 <widget class="QDialog" name="TagEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>229</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Tags</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tagDescripText">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>70</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Brucellosis Tag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagDescrip">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="burcellosisTagEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Q3ListBox" name="tagListBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>Q3ListBox::Multi</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tagEdit</tabstop>
  <tabstop>burcellosisTagEdit</tabstop>
  <tabstop>tagDescrip</tabstop>
  <tabstop>tagListBox</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/tagdao.h</include>
  <include location="local">types/customer.h</include>
  <include location="local">types/animalcode.h</include>
  <include location="local">types/transaction.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
